server/bin/pbench-copy-sosreports.sh

```diff
@@ -138,6 +138,7 @@ for result in $list ;do
 
     if [[  $nsr == 0 ]] ;then
         echo "$TS: No sosreports found for $result"
+	res="${result}.Bad-Hostname"
```

Comment:
Not sure what the intention here is, but if it is to make a change on disk for the result tar ball name, that won't really work.